README.md

```diff
@@ -4,7 +4,7 @@ Code for <https://eff.org/privacybadger>.
 
 ## Development
 
-1. Install the extended version of [Hugo](https://gohugo.io/getting-started/installing/), eg `snap install hugo --channel=extended` on Linux.
+1. Install the extended version of [Hugo](https://gohugo.io/getting-started/installing/) v0.55.3, eg `snap install hugo --channel=extended` on Linux. Installing the most recent version of Hugo will result in some unexpected behavior when mounting a development build of the site.
```

Comment:
What's the "source of truth" for the version of Hugo we should use? Is it the Docker file or the Travis YAML? We should link to that instead of hardcoding the version in another place (that can drift out of sync).

Reply:
Does `snap install hugo --channel=extended` install the latest version? We should amend or remove the example command so that it's not misleading.

Reply:
The Dockerfile should always build a working site for production, so that's a good source of truth. `snap install hugo --channel=extended` does not install a compatible version of hugo. As far as I can tell, snap does not allow us to select a specific version -- I got my development build working by fetching the right `.deb` from the [github release](https://github.com/gohugoio/hugo/releases/tag/v0.55.3).

Maybe for now we could remove the snap command and keep v0.55.3 documented in the README, and meanwhile I will work on getting the site built with the latest release. When that's done we can keep an eye on how much breakage there is between versions at this point, and then consider reverting the README to include the snap command?

Reply:
Removing the snap command (specific Hugo installation instructions) sounds fine to me.

I suggest linking to the Dockerfile on the master branch for the exact version to install. We can link to the Hugo release for v0.55.3 too, I just want it to be clear that the Dockerfile on master is the source of truth.

As long as there is potential for breakage between Hugo versions, it seems preferable to stick to using a specific as opposed to the latest version.

Reply:
I've done some work on this in #59.